zerver/lib/name_restrictions.py

```diff
@@ -173,6 +174,7 @@ def is_disposable_domain(domain: str) -> bool:
     "cpanel",
     "css",
     "cssproxy",
+    "customers",
```

Comment:
We check if trimming a trailing `s` is in the list, so making this `customer` is the more general form.
```suggestion
    "customer",
```